spec/features/user_add_new_review_spec.rb

```diff
@@ -7,7 +7,7 @@
 } do
   scenario 'navigate to new review form' do
     user = FactoryBot.create(:user, username: "testuser")
-    trail = Trail.create!(name: 'First Trail', street: '123 Maple St.', city: 'Boston', state: 'MA', zip: '12345', start_latitude: '1', start_longitude: '2', length: '6', difficulty: '7', elevation: '80', user_id: user)
+    trail = Trail.create!(name: 'First Trail', street: '123 Maple St.', city: 'Boston', state: 'MA', zip: '12345', start_latitude: '1', start_longitude: '2', length: '6', difficulty: '7', elevation: '80', created_by: user.id)
 
```

Comment:
If you are using FactoryBot for your users, you should be using it for your trails. Please update.